### PR TITLE
Dynamic iters default chunk size

### PR DESCRIPTION
**Pull request by s-geronimoanderson**
Added default value 1 for dynamic iterator's chunkSize formal, and updated shootouts not to pass 1 as the chunkSize actual. Note: My editor seems to have removed trailing spaces automatically, which makes the diff appear more involved than it actually is.

### CODE REVIEW

**Comment by benharsh** [comment]
These changes look good to me. Since this PR modifies the release shootouts, @chapel-lang/perf-team might be interested in reviewing as well.

**Comment by daviditen** [comment]
In mandelbrot.chpl and mandelbrot-fast.chpl `chunkSize` was a user-configurable setting, but now it's always forced to 1.  I think it should probably be put back in those two cases.

**Comment by bradcray** [comment]
> In mandelbrot.chpl and mandelbrot-fast.chpl chunkSize was a user-configurable setting, but now it's always forced to 1. I think it should probably be put back in those two cases.

I agree.  Or we could remove the config if we don't anticipate values other than 1 being useful...

File: test/release/examples/benchmarks/shootout/mandelbrot-fast.chpl [hunk 1]
```diff
@@ -10,7 +10,7 @@ use DynamicIters;
 
 config const n = 200,             // image size in pixels (n x n)
              maxIter = 50,        // max # of iterations per pixel
-             limit = 4.0,         // per-pixel convergence limit
+             limit = 4.0;         // per-pixel convergence limit
```
Comment (benharsh) [inline test/release/examples/benchmarks/shootout/mandelbrot-fast.chpl]: I think you need to replace the semicolon with a comma here.

File: test/release/examples/benchmarks/shootout/mandelbrot.chpl [hunk 2]
```diff
@@ -10,7 +10,7 @@ use DynamicIters;
 
 config const n = 200,              // image size in pixels (n x n)
              maxIter = 50,         // max # of iterations per pixel
-             limit = 4.0,          // per-pixel convergence limit
+             limit = 4.0;          // per-pixel convergence limit
```
Comment (benharsh) [inline test/release/examples/benchmarks/shootout/mandelbrot.chpl]: And here

**Comment by s-geronimoanderson** [comment]
I debated what to do in the mandelbrot* tests, but I also agree that it's sensible to leave the chunkSize config const as intended. I've added it back in for those.

**Pull request merged**